packages/react/src/Section/Section.module.css

```
@@ -0,0 +1,156 @@
+  max-width: none;
+}
+
+.Section--rounded {
+  border-radius: var(--rounded-radius) var(--rounded-radius) 0 0;
+  margin-top: calc(-1 * var(--rounded-radius));
+}
+
+/* A progressive enhancment to adjust block-end padding to account for adjacent section rounded corners overlap. */
+.Section:has(+ .Section--rounded) {
+  --rounded-padding-overlap-adjustment: var(--rounded-radius);
+}
+
+.Section--paddingBlockStart-none,
+.Section-narrow--paddingBlockStart-none {
+  padding-block-start: 0;
+}
+
+.Section--paddingBlockStart-condensed,
+.Section-narrow--paddingBlockStart-condensed {
+  padding-block-start: var(--brand-Section-padding-condensed);
+}
+.Section--paddingBlockStart-normal,
+.Section-narrow--paddingBlockStart-normal {
+  padding-block-start: var(--brand-Section-padding-normal);
+}
+.Section--paddingBlockStart-spacious,
+.Section-narrow--paddingBlockStart-spacious {
+  padding-block-start: var(--brand-Section-padding-spacious);
+}
+
+.Section--paddingBlockEnd-none,
+.Section-narrow--paddingBlockEnd-none {
+  --padding-block-end: 0px;
+}
+
+.Section--paddingBlockEnd-condensed,
+.Section-narrow--paddingBlockEnd-condensed {
+  --padding-block-end: var(--brand-Section-padding-condensed);
+}
+.Section--paddingBlockEnd-normal,
+.Section-narrow--paddingBlockEnd-normal {
+  --padding-block-end: var(--brand-Section-padding-normal);
+}
+.Section--paddingBlockEnd-spacious,
+.Section-narrow--paddingBlockEnd-spaciou {
+  --padding-block-end: var(--brand-Section-padding-spacious);
+}
+
+@media screen and (min-width: 40rem) {
+  .Section {
+    background-color: var(--brand-Section-regular-background-color, var(--brand-Section-background-color));
+    background-image: var(--brand-Section-regular-background-image-src, var(--brand-Section-background-image-src));
+    background-size: var(--brand-Section-regular-background-image-size, var(--brand-Section-background-image-size));
+    background-position: var(
+      --brand-Section-regular-background-image-position,
+      var(--brand-Section-background-image-position)
+    );
+  }
+
+  .Section-regular--paddingBlockStart-none {
+    padding-block-start: 0;
+  }
+
+  .Section-regular--paddingBlockStart-condensed {
+    padding-block-start: var(--brand-Section-padding-condensed);
+  }
+  .Section-regular--paddingBlockStart-normal {
+    padding-block-start: var(--brand-Section-padding-normal);
+  }
+  .Section-regular--paddingBlockStart-spacious {
+    padding-block-start: var(--brand-Section-padding-spacious);
+  }
+
+  .Section-regular--paddingBlockEnd-none {
+    --padding-block-end: 0px;
+  }
+
+  .Section-regular--paddingBlockEnd-condensed {
+    --padding-block-end: var(--brand-Section-padding-condensed);
+  }
+  .Section-regular--paddingBlockEnd-normal {
+    --padding-block-end: var(--brand-Section-padding-normal);
+  }
+  .Section-regular--paddingBlockEnd-spacious {
+    --padding-block-end: var(--brand-Section-padding-spacious);
+  }
+}
+
+@media screen and (min-width: 63.25rem) {
```

To match our usual `wide` breakpoint...

```suggestion
@media screen and (min-width: 80rem) {
```